### PR TITLE
feat: add functionality to disable timer

### PR DESCRIPTION
**Pull request by olastor**

### CODE REVIEW

File: src/app/meditation/meditation.component.ts [hunk 1]
```diff
@@ -30,6 +30,7 @@ export class MeditationComponent {
 
   // user profile
   profile;
+  disableTimer: Boolean = false;
```
Comment (binarious) [inline src/app/meditation/meditation.component.ts]: `Boolean` -> `boolean`

**Comment by binarious** [comment]
Why did you add this? Generally disabling the timer?

**Comment by olastor** [comment]
Did I misinterpret #194?

('Couldn't we put a form in the user profile?', 'I would prefer a form in the profile, too.')

**Comment by binarious** [comment]
A form for logging a session. Input fields with `walking`, `sitting`, `date` and `time`.

**Pull request closed without merging**